app/cdap/components/ImportDataset/index.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useEffect, useState, useContext } from 'react';
+import Box from '@material-ui/core/Box';
+import { Button } from '@material-ui/core';
+import { useStyles } from './styles';
+import DrawerWidget from 'components/DrawerWidget';
+import DatasetBody from './Components/ImportDatasetBody';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import { ConnectionsContext } from 'components/Connections/ConnectionsContext';
+import UploadFile from 'services/upload-file';
+import isNil from 'lodash/isNil';
+import Cookies from 'universal-cookie';
+import { Redirect } from 'react-router';
+import PositionedSnackbar from 'components/SnackbarComponent';
+import T from 'i18n-react';
+
+const FILE_SIZE_LIMIT = 10 * 1024 * 1024; // 10 MB
+const cookie = new Cookies();
+
+export default function(props) {
```

Comment:
if we are not using props then remove it

Reply:
It is used in line 50 .
 props.handleClosePanel();

Reply:
Made an interface .